src/include/storage/predicate/column_predicate.h

```diff
@@ -20,7 +20,7 @@ class KUZU_API ColumnPredicateSet {
     }
     bool isEmpty() const { return predicates.empty(); }
 
-    common::ZoneMapCheckResult checkZoneMap(const CompressionMetadata& metadata);
+    common::ZoneMapCheckResult checkZoneMap(const ColumnChunkStats& metadata) const;
```

Comment:
Can u make the class name and variable name be consistent? I think you'd want to rename `metadata` as `stats`.

Reply:
Sure I'll add this in a future PR.